docs/general/events.md

```diff
@@ -43,18 +33,12 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
-
-**[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
-
-`event.domain` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+| `event.name` | string | Uniquely identifies the event. | `browser.mouse.click`; `browser.exception` | Required |
 
-| Value  | Description |
-|---|---|
-| `browser` | Events from browser apps |
-| `device` | Events from mobile apps |
-| `k8s` | Events from Kubernetes |
+Event names are subject to the same rules as [attribute names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/common/attribute-naming.md).
```

Comment:
Should we not point to a specific version of the specification?

Reply:
No, I don't think we should. Spec is versioned all together, all of its documents. If the linked document is updated it should apply to this doc too.

Reply:
The specs are linked to in a lot of other docs in this same manner. Should we update all of them and is that necessarily an ask or work that is outside the scope of this PR?

Reply:
No need to update other links in this PR.

I wonder if any reviewer has any arguments in favour of linking to a specific version? I thought it is a mistake but since it is in many places perhaps it is deliberate and in that case I wonder what the reason is.

Reply:
Linking to a specific version will be confusing unless the version is part of the link text and is specifically called out.

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/549 opened to update the other links in the repo.